pennylane/tape/tape.py

```diff
@@ -441,15 +444,6 @@ def stop_recording(self):
     def _process_queue(self):
         """Process the annotated queue, creating a list of quantum
         operations and measurement processes.
-
```

Comment:
Maybe instead of fully deleting, we can keep some sort of comment like:
```
Sets ``_ops``, ``_prep``, ``_measurements`` and calls ``_update``.  ``_update`` then sets a large number of properties.

```